examples/github-oauth/configuration/groovy/set-url.groovy

```diff
@@ -0,0 +1,5 @@
+import jenkins.model.JenkinsLocationConfiguration
+
+def jlc = JenkinsLocationConfiguration.get()
+jlc.setUrl("http://127.0.0.1:8080")
```

Comment:
Possibly for this example, if the user is deploying an Ingress, this URL should be set to the Ingress's domain name. Maybe this can be done via environment variable?

Reply:
I will do this as a separate PR I think.